src/modules/learning-object-service/fileManager/FileManagerController.ts

```diff
@@ -45,14 +39,48 @@ export class FileManagerController implements Controller {
      *              type: string
      *              format: binary
      *      401:
-     *        description: UNAUTHENTICATED - User is not logged in
+     *        description: UNAUTHENTICATED - Invalid access. You must have a valid token to access this resource.
      *      403:
-     *        description: UNAUTHORIZED - User is trying to access an in review object as a unprivileged user
+     *        description: UNAUTHORIZED - User {username} does not have access to download the requested Learning Object
      *      404:
-     *        description: NOT FOUND - Object not found
+     *        description: NOT FOUND - Bundle could not be found, Author of learning object not found, Learning Object not found
      */
-    router.route('/users/:username/learning-objects/:cuid/versions/:version/bundle').get(this.proxyLearningObjectRequest((req: Request) => `/users/${encodeURIComponent(req.params.username)}/learning-objects/${encodeURIComponent(req.params.cuid)}/versions/${encodeURIComponent(req.params.version)}/bundle`));
+    router.route('/users/:username/learning-objects/:id/bundle').get(this.proxyLearningObjectRequest((req: Request) => `/users/${encodeURIComponent(req.params.username)}/learning-objects/${encodeURIComponent(req.params.id)}/bundle`));
     
+    /**
+     * @swagger
+     * /users/{username}/learning-objects/{id}/bundle:
+     *  post:
+     *    description: Bundles a learning object
+     *    tags:
+     *      - Learning Object Service
+     *    parameters:
+     *      - in: path
+     *        name: username
+     *        schema:
+     *            type: string
+     *        required: true
+     *        description: The username of the author
+     *      - in: path
+     *        name: id
+     *        schema:
+     *            type: string
+     *        required: true
+     *        description: The id of the object to bundle
+     *     responses:
```

Comment:
I pulled the branch and the indentation is off here. The dev environment threw an error "YAML Bad Mapping entry"